src/Appwrite/Platform/Modules/Functions/Workers/Builds.php

```diff
@@ -713,6 +720,107 @@ protected function buildDeployment(Device $deviceForFunctions, Func $queueForFun
 
             Console::success("Build id: $buildId created");
 
+            if ($resource->getCollection() === 'sites') {
+                try {
+                    $rule = Authorization::skip(fn () => $dbForPlatform->findOne('rules', [
+                        Query::equal("projectInternalId", [$project->getInternalId()]),
+                        Query::equal("resourceType", ["deployment"]),
+                        Query::equal("resourceInternalId", [$deployment->getInternalId()])
+                    ]));
+
+                    if ($rule->isEmpty()) {
+                        throw new \Exception("Rule for build not found");
+                    }
+
+                    $client = new FetchClient();
+                    $client->addHeader('content-type', FetchClient::CONTENT_TYPE_APPLICATION_JSON);
+
+                    $bucket = $dbForPlatform->getDocument('buckets', 'screenshots');
+
+                    $configs = [
+                        'screenshot' => [
+                            'headers' => [ 'x-appwrite-hostname' => $rule->getAttribute('domain') ],
+                            'url' => 'http://traefik/?appwrite-preview=1&appwrite-theme=light',
+                            'color' => 'light'
```

Comment:
Should it be called `theme` here as well?

Reply:
Hmm, it's called `color` on Appwrite Browser because internally it sets `colorScheme`, and to simplify, just `color`.

But I agree with you, in contect of Browser API params, `theme` would make more sense.